docs/team/tau-bar.md

```diff
@@ -18,47 +20,67 @@ Given below are my contributions to the project.
-  2. Added the ability to delete friends.
+  2. Added the ability to delete friends in [PR#48](https://github.com/AY2122S1-CS2103T-W13-4/tp/pull/48/files).
      - What it does: allows the user to delete a friend from gitGud using the friend's unique `FRIEND_ID`. `Refactor - Enhancement`
      - Justification: This feature improves the product significantly because a user can now delete a friend from
               gitGud in the event that they made a mistake in the details or simply wishes to remove a friend.
-  3. Added the ability to delete games.
+  3. Added the ability to delete games in [PR#68](https://github.com/AY2122S1-CS2103T-W13-4/tp/pull/68).
      - What it does: allows the user to delete a game from gitGud using the game's unique `GAME_ID`. `New Feature` 
      - Justification: This feature improves the product significantly because a user can easily delete a game from
          gitGud in the event that they made a mistake in the details or simply wishes to remove a game because the 
        user no longer plays the game.
-  4. Restructured and styled the UI to be in its current state with 2 lists, the friend list and the game list, as well 
-     as a 
-     main card in the middle to display the requested information by the user.
+  4. Restructured and styled the UI for v1.2, v1.3 and v1.4 to be in its current state with 2 lists, the friend list 
+     and the game list, as well as a main card in the middle to display the requested information by the user in 
+  [PR#145](https://github.com/AY2122S1-CS2103T-W13-4/tp/pull/145).
      - What it does:  allows the user to see details about a friend or a game when the user runs the `friend --get` 
        or `game --get` command.`New Feature`
      - Justification: This feature improves the product significantly, as one of the purposes of gitGud is to be 
        able to store and view your friends' data easily, such as their in-game usernames as well as their skill levels, 
        as well as the friend's availabilities, to make it easy for the user to choose who to play what game at what time.
          
 * **Testing**
-  1. Adapted the previous tests for our implementation of AddFriendCommand and DeleteFriendCommand.
-  2. Added tests for DeleteGameCommand and DeleteGameCommandParser.
-  3. Added tests for LinkFriendCommand and UnlinkFriendCommand, and linking methods in other classes.
+  1. Adapted the previous tests for our implementation of AddFriendCommand and DeleteFriendCommand in 
+  [PR#48](https://github.com/AY2122S1-CS2103T-W13-4/tp/pull/48/files).
+  2. Added unit/integration tests for DeleteGameCommand and DeleteGameCommandParser in 
+  [PR#68](https://github.com/AY2122S1-CS2103T-W13-4/tp/pull/68)
+  3. Added unit/integration tests for LinkFriendCommand and UnlinkFriendCommand, and linking methods in other 
+     classes in [PR#184](https://github.com/AY2122S1-CS2103T-W13-4/tp/pull/184).
```

Comment:
Consider standardising the URLs? Some lead to files while others lead to the description. Personally, I think it will be better if the URL leads the reader to the main description of the URL rather than files, for some.